src/App.css

```diff
@@ -160,3 +169,79 @@
 .d2 > * {
   margin-bottom: 0.5rem;
 }
+
+
+
+
+@media screen and (max-width: 800px) {
+  .navmargin {
+    margin-left: 200px;
+    margin-top: 500px;
+  }
+
+  .d2 > * {
+    margin-bottom: 1rem;
+  }
+
+  .input-block {width:250px;}
+
+  .custom-button {
+    width: 250px; 
+  }
+
+  .custom-select__menu {
+    width: 250px !important; 
+  }
+  .custom-select__control {
```

Comment:
avoid using important, correct your logic here

Reply:
Is over ridding one.